dvc/command/ls/__init__.py

```diff
@@ -72,6 +84,11 @@ def add_parser(subparsers, parent_parser):
     list_parser.add_argument(
         "--show-json", action="store_true", help="Show output in JSON format."
     )
+    list_parser.add_argument(
+        "--show-pretty-json",
```

Comment:
We could just prettify the JSON by default by checking if we are in `isatty` mode and then prettify that output.

Reply:
Thanks for the suggestion! The patch was updated